src/common/browser-version/ios-version.js

```diff
@@ -4,7 +4,8 @@ export const isiOS = /(iPad|iPhone|iPod)/g.test(navigator.userAgent)
 
 // Shared Web Workers introduced in iOS 16.0+ and n/a in 15.6-
 export const iOS_below16 = isiOS && Boolean(typeof SharedWorker === 'undefined')
-/* ^ It was discovered in Safari 14 (https://bugs.webkit.org/show_bug.cgi?id=225305) that the buffered flag in PerformanceObserver
-	did not work. This affects our onFCP metric in particular since web-vitals uses that flag to retrieve paint timing entries.
-	This was fixed in v16+.
+/**
```

Comment:
What's with this? The double asterisks are associated with JSDocs and will become the description of whatever follows it, whereas the single asterisks are for multi-line comments and don't get auto tagged as jsdoc